scripts/clean-and-init.sh

```diff
@@ -36,6 +36,7 @@ redis-cli flushall
 # Init and create db and indexes
```

Comment:
can you also change here from db destroy to db drop? We just want to drop the tables, no need to destroy the db, and then we can use same script for dev and deployment ;)